### PR TITLE
Fix Website Display

### PR DESCRIPTION
**Pull request by BrigetteDoelp**
### Description
Fixes the broken display and style linkage.
### Motivation and Context
We couldn't test for functinoality or do aria testing and tags without hte website display being vaseline. There is currently no functionality, but I am pushing and merging this so that my team partners can work on their respoctive tasks with a proper website UI.
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by Estaffieri** [comment]
I am concerned about the +12,474 lines of code being added here? Are we including something that shouldn't be merged?

File: src/domUpdates.js [hunk 1]
```diff
@@ -16,10 +16,16 @@ let domUpdates = {
...
+  return words.split(" ").map(word => {
+    return word.charAt(0).toUpperCase() + word.slice(1);
+  }).join(" ");
+},
+
  listTags(allTags) {
   allTags.forEach(tag => {
     let tagHtml = `<li><input type="checkbox" class="checked-tag" id="${tag}">
-      <label for="${tag}">${capitalize(tag)}</label></li>`;
+      <label for="${tag}">${this.capitalize(tag)}</label></li>`;
```
Comment (MichaelEWalker87) [inline src/domUpdates.js:28]: good catch that it needs a this

File: src/index.html [hunk 2]
```diff
@@ -3,10 +3,16 @@
   <head>
     <meta charset="utf-8">
     <title>Fresh Picks</title>
-    <link rel="stylesheet" type="text/css" href="../styles.css">
+    <link rel="stylesheet" type="text/css" href="./css/styles.scss">
```
Comment (MichaelEWalker87) [inline src/index.html:6]: Nice that the file path had to be one of the main errors we were getting

File: src/index.js [hunk 3]
```diff
@@ -5,9 +5,17 @@
...
 
 // An example of how you tell webpack to use an image (also need to link to it in the index.html)
-import './images/turing-logo.png'
+import './images/apple-logo-outline.png'
+import './images/apple-logo.png'
+import './images/cookbook.png'
+import './images/seasoning.png'
+import './images/pancakes.png'
+import './images/search.png'
+import './images/green-apples.png'
```
Comment (MichaelEWalker87) [inline src/index.js:17]: now that these images are fixed I have a better idea on the spacing!

**Review by MichaelEWalker87 — APPROVED** [review]

**Review by MichaelEWalker87 — CHANGES_REQUESTED** [review]
might need the gitignore file

**Comment by Estaffieri** [comment]
Looks like we've got that gitignore included now. I am merging this request! We can come back to this point if everything breaks!

**Pull request merged**